build.sh

```diff
@@ -215,6 +215,13 @@ build_native()
             fi
         fi
 
+        echo "Restoring the OptimizationData package"
```

Comment:
Like I've said for the build.cmd, it would be great to add a command line option to build.cmd that would allow skipping this.